Gemfile

```diff
@@ -10,7 +10,7 @@ def vanagon_location_for(place)
   end
 end
 
-gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '0.11.3')
+gem 'vanagon', *vanagon_location_for(ENV['VANAGON_LOCATION'] || '~> 0.12.1')
```

Comment:
Won't this just continue to act in the same way (where it doesn't auto update) because we specified the Z release ?